### PR TITLE
Test build on multiple Ruby versions

### PR DESCRIPTION
**Pull request by nicolasleger**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,4 +3,7 @@ cache: bundler
 sudo: false
 
 rvm:
-  - 2.3.1
+  - '2.2'
```
Comment (kevinhughes27) [inline .travis.yml]: I seem to recall we removed support for ruby 2.2 a while back.

**Comment by kevinhughes27** [comment]
:+1:

File: .travis.yml (hunk 1 above)
Comment (nicolasleger) [reply .travis.yml]: Thanks, ok I remove it.

**Comment by nicolasleger** [comment]
@kevinhughes27 build now fixed

**Comment by jamiemtdwyer** [comment]
Thanks @nicolasleger !

**Pull request merged**